diagnostic.md

```diff
@@ -5,74 +5,83 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```md
-// your response here
+The backend allows for persistence of data and it allows us to validate everything coming from the client.
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```md
-// your response here
+Model
 ```
 
 Which layer in the MVC pattern communicates with the model?
 
 ```md
-// your response here
+Controller
 ```
 
 Why don't we use views in our interpretation of the MVC pattern?
 
 ```md
-// your response here
+We do not use views because we are building an API, not a web app.
 ```
 
 What does C.R.U.D stand for?
 
 ```md
-// your response here
+Create, Read, Update, Destroy
 ```
 
 In which part of the MVC pattern can we find C.R.U.D actions?
 
 ```md
-// your response here
+I'm slightly confused by this question.
+
+The controller calls the CRUD actions but the model is where the CRUD methods themselves are.
 ```
 
 List at least 5 standard rails actions that C.R.U.D requests correspond to?
 
 ```md
-// your response here
+#index
+#show
+#update
+#create
+#destroy
 ```
 
 A user action fires a `GET` request for `/people/1`. Explain in detail each step
 required for data to be returned to the client. (bullet points or ordered list)
 
 ```md
-// your response here
+- The router parses the http request
+- The router calls the appropriate controller action
+- The controller consults the model to get the required data
+- The controller responds to the client with the requested data
 ```
```

Comment:
Good job! Here's a little more detail that may clear up some of the gaps of detail there.